content/en/user-manual/scripting/script-attributes.md

```diff
@@ -138,4 +138,33 @@ MyScript.attributes.add('value', {
 
 Use the enum property to declare the list of possible values for your enumeration. Property is an array of objects where each object is an option where `key` is a title of an option and `value` is a value for attribute. This property can be used for various attribute types, e.g. `number`, `string`, `vec3`.
 
+
+### JSON attribute
+
+The JSON attribute allows you to create nested attributes of the other attribute types. For every JSON attribute you must specify a schema to describe its properties. The schema contains other regular script attribute definitions like above. For example:
+
+```javascript
+MyScript.attributes.add('gameConfig', {
+    type: 'json',
+    schema: [{
+        name: 'numEnemies',
+        type: 'number',
+        default: '10'
```

Comment:
Remove the `'` as it's for a number type. ie:
```
        name: 'numEnemies',
        type: 'number',
        default: 10
```

Reply:
Fixed